frontend/src/components/CANs/CANBudgetForm/CANBudgetForm.jsx

```diff
@@ -56,12 +55,6 @@ const CANBudgetForm = ({
                 className="usa-button usa-button--outline margin-top-4"
                 data-cy="add-fy-budget"
             >
-                <svg
```

Comment:
think we only want to show the + icon when we add but not when we update

Reply:
@fpigeonjr fixed, enjoy.   FYI @kimschulke